requirements.txt

```diff
@@ -11,7 +11,7 @@ rospkg==1.2.9
 pathlib==1.0.1
 jderobot-jderobottypes==1.0.0
 catkin_pkg==0.4.23
-numpy==1.22.0
+numpy
```

Comment:
Please move back to the selected versions for these modules. We have already updated this file in the noetic-devel branch